python/lsst/ts/mthexapod/hexapod_csc.py

```diff
@@ -282,8 +282,12 @@ def __init__(
         )
 
         # See the ts_config_ocs/ESS
+        # 120 is the offset defined in ts_config_ocs/ESS/v8/_init.yaml
```

Comment:
There is no offset defined in the ESS config. The ESS indices simply are 120 and 121.

Reply:
Improved in the commit: 6c1f54a. Thanks!